runtime/compilers/rillv1/parser.go

```diff
@@ -524,6 +524,12 @@ func (p *Parser) parsePaths(ctx context.Context, paths []string) error {
 		if pathIsRillYAML(b) {
 			return 1
 		}
+		if pathIsDotEnv(a) {
+			return -1
+		}
+		if pathIsDotEnv(b) {
+			return 1
+		}
```

Comment:
Why does `.env` need to be parsed before other resources? I don't think we read variables from inside the parser.

The reason `rill.yaml` is parsed first is because you can specify resource defaults in `rill.yaml` that are applied when parsing other files.

Reply:
Yeah I did these changes to test something else and realised we don't read variables in parser but missed removing these changes. Will remove.